### DISCUSSION TITLE
How to Configure Custom Endpoint URL for API in Autogen?

### DISCUSSION THREAD

**Discussion by gopinathselvarajan** [microsoft/autogen#4419] [Q&A]
I have a doubt. In Autogen, we are supposed to pass the OpenAI API key through a configuration setup. However, in my organization, they have set up a tenant and a base URL to access the API keys. The API responds based on the endpoint URL provided. In this case, how can I configure my organization's endpoint URL to work with Autogen?

**Comment by ekzhu** [comment]
If the endpoint is OpenAI but just through a proxy, you can use `base_url` parameter to set your corporate proxy endpoint. https://microsoft.github.io/autogen/dev/reference/python/autogen_ext/autogen_ext.models.html#autogen_ext.models.OpenAIChatCompletionClient
